starknet_devnet/starknet_wrapper.py

```diff
@@ -230,15 +228,17 @@ async def store_transaction(self, contract_address: str, status: TxStatus, error
                 "tx_id": new_id
             }
         else:
-            block_hash, block_number = await self.generate_block(transaction["transaction"])
+            block_hash, block_number = await self.__generate_block(transaction["transaction"])
             transaction["block_hash"] = block_hash
             transaction["block_number"] = block_number
 
-        self.transactions.append(transaction)
+        self.__transactions.append(transaction)
         return hex_new_id
 
+    # pylint: disable=too-many-arguments
```

Comment:
Maybe the argument could be a single Object/Dataclass with all these arguments as properties